tests/platform_tests/test_platform_info.py

```diff
@@ -7,6 +7,7 @@
 import json
 import logging
 import time
+from retry.api import retry_call
```

Comment:
It looks like a new dependency is introduced here: https://pypi.org/project/retry/
Is there a way to avoid this dependency?
If not, can you update the sonic-mgmt docker file to include installation of this retry package?

Reply:
Oh, I didn't notice that it has been added to the sonic-mgmt image in PR https://github.com/Azure/sonic-buildimage/pull/7997. LGTM then.